ghu_web/ghu_global/static/ghu_global/css/style.css

```diff
@@ -14,6 +14,7 @@ body {
     margin: 0px;
     padding: 4px;
     text-align: right;
+    background-color: #00e673
```

Comment:
Don't have any hardware to check this out, but dang, this seems like a pretty strong color for the background. Am I wrong? Will the text have a white background at least?